hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/HddsVolume.java

```diff
@@ -350,6 +368,63 @@ public void createDbStore(MutableVolumeSet dbVolumeSet) throws IOException {
     }
   }
 
+  /**
+   * Ensure that volume is initialized properly with
+   * cleanup path. Should disk be re-inserted into
+   * cluster, cleanup path should already be on
+   * disk. This method syncs the HddsVolume
+   * with the path on disk.
+   * 
+   * @param id clusterId or scmId
+   */
+  public void checkTmpDirPaths(String id) {
```

Comment:
With restructuring the location of configuring the tmp directory into the `HddsVolume`, passing the `clusterID` to the tmp dir path config is redundant.  _Remove_ all interfaces with `id `(clusterid) as an argument as the `clusterID `is a private variable in the `HddsVolume `(`StorageVolume`) instance (use `getClusterID`).